addons/delivery/models/delivery_price_rule.py

```diff
@@ -4,6 +4,7 @@
 
 from odoo.tools import format_amount
 
+
```

Comment:
Non-necessary diff.

Reply:
Hello @vava-odoo changes have been done please check thank you.